<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="single_arm_v12">
  <link name="dummy"/>
  <joint name="dummy_joint" type="fixed">
    <parent link="dummy"/>
    <child link="base_link"/>
    </joint>
  <link name='base_link'>
  <!-- <link
    name="base_link"> -->
    <inertial>
      <origin
        xyz="0.00015625 0.00020296 0.064397"
        rpy="0 0 0" />
      <mass
        value="0.47711" />
      <inertia
        ixx="0.00059036"
        ixy="3.555E-10"
        ixz="4.1821E-06"
        iyy="0.00058997"
        iyz="9.8398E-08"
        izz="0.00022102" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://single_arm_v12/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://single_arm_v12/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="Link1">
    <inertial>
      <origin
        xyz="3.4484E-06 0.00039863 0.028769"
        rpy="0 0 0" />
      <mass
        value="0.38613" />
      <inertia
        ixx="0.00047431"
        ixy="-7.0827E-08"
        ixz="-3.5556E-10"
        iyy="0.00017465"
        iyz="4.1821E-06"
        izz="0.00048907" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://single_arm_v12/meshes/Link1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://single_arm_v12/meshes/Link1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint1"
    type="revolute">
    <origin
      xyz="0 0 0.1445"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="Link1" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="50"
      velocity="0" />
  </joint>
  <link
    name="Link2">
    <inertial>
      <origin
        xyz="0.283433030970685 -0.000330697518582679 0.0187199245749208"
        rpy="0 0 0" />
      <mass
        value="0.478163838060215"  />
      <inertia
        ixx="0.00111448622362089"
        ixy="-0.0000370988484703014"
        ixz="-0.00118065544458626"
        iyy="0.00685193924710543"
        iyz="-0.0000155267735632356"
        izz="0.00596695471314002" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://dynamics/src/dynamics/meshes/merged_single_arm_axis_2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.89804 0.91765 0.92941 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://dynamics/src/dynamics/meshes/merged_single_arm_axis_2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint2"
    type="revolute">
    <origin
      xyz="0 -0.0685 0.0315"
      rpy="1.5708 0 0" />
    <parent
      link="Link1" />
    <child
      link="Link2" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="50"
      velocity="0" />
  </joint>
  <link
    name="Link3">
    <inertial>
      <origin
        xyz="0.289897056562879 0.000220585774777601 -0.0177189470022211"
        rpy="0 0 0" />
      <mass
        value="0.441060161173263"  />
      <inertia
        ixx="0.000800686662611200"
        ixy="0.0000237251498077782"
        ixz="0.000742345796173437"
        iyy="0.00625559935973774"
        iyz="-0.00000891007317366508"
        izz="0.00566719543969895" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://dynamics/src/dynamics/meshes/merged_single_arm_axis_3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.898039215686275 0.917647058823529 0.929411764705882 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://dynamics/src/dynamics/meshes/merged_single_arm_axis_3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint3"
    type="revolute">
    <origin
      xyz="0.369235515941162 0 -0.0325"
      rpy="0 0 0" />
    <parent
      link="Link2" />
    <child
      link="Link3" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="50"
      velocity="0" />
  </joint>
  <link
    name="Link4">
    <inertial>
      <origin
        xyz="-1.11022302462516E-16 0.0124471610012716 0.0305494763505743"
        rpy="0 0 0" />
      <mass
        value="0.313854171959486" />
      <inertia
        ixx="0.00030905374217572"
        ixy="2.20104937481116E-19"
        ixz="1.99389141092033E-20"
        iyy="0.000138417182248756"
        iyz="-3.51429818496687E-06"
        izz="0.000324741574348905" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://single_arm_v12/meshes/Link4.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.898039215686275 0.917647058823529 0.929411764705882 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://single_arm_v12/meshes/Link4.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint4"
    type="revolute">
    <origin
      xyz="0.378764484058838 0 0.0325"
      rpy="0 0 3.1416" />
    <parent
      link="Link3" />
    <child
      link="Link4" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="50"
      velocity="0" />
  </joint>
  <link
    name="Link5">
    <inertial>
      <origin
        xyz="0.0305494763505902 0.0124160882702156 -0.000878959076247798"
        rpy="0 0 0" />
      <mass
        value="0.313854171959487" />
      <inertia
        ixx="0.000324741574348905"
        ixy="-3.50552519309465E-06"
        ixz="2.48162957481373E-07"
        iyy="0.000139268063164469"
        iyz="1.20194589741801E-05"
        izz="0.000308202861260008" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://single_arm_v12/meshes/Link5.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.898039215686275 0.917647058823529 0.929411764705882 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://single_arm_v12/meshes/Link5.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint5"
    type="revolute">
    <origin
      xyz="0 0.0685 0.0346"
      rpy="1.5708 0 1.5708" />
    <parent
      link="Link4" />
    <child
      link="Link5" />
    <axis
      xyz="-1 0 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="50"
      velocity="0" />
  </joint>
  <link
    name="Link6">
    <inertial>
      <origin
        xyz="-1.11022302462516E-16 1.73472347597681E-18 0.0055"
        rpy="0 0 0" />
      <mass
        value="0.0173494454294496" />
      <inertia
        ixx="2.53988652151547E-06"
        ixy="-2.00729161979554E-22"
        ixz="-3.90838944351189E-22"
        iyy="2.53988652151547E-06"
        iyz="-1.17534537510526E-22"
        izz="4.79061561920678E-06" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://single_arm_v12/meshes/Link6.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://single_arm_v12/meshes/Link6.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint6"
    type="revolute">
    <origin
      xyz="0.0346 0.0685 0"
      rpy="1.5708 -1.5708 0" />
    <parent
      link="Link5" />
    <child
      link="Link6" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="50"
      velocity="0" />
  </joint>
</robot>